tests/test_pipelines.py

```diff
@@ -6,6 +6,7 @@
 from pytest_lazyfixture import lazy_fixture
 
 from pangeo_forge_recipes.executors.base import Pipeline, Stage
+from pangeo_forge_recipes.executors.beam import BeamPipelineExecutor
```

Comment:
Since beam will be an optional dependency, we need some sort of [importorskip](https://docs.pytest.org/en/latest/reference/reference.html#pytest-importorskip-ref) logic in this test.

We don't want to skip the whole module if beam is not importable--just the beam part.

Reply:
TIL about this pytest feature! I'll update the tests.

Reply:
I ended up taking a different approach to achieve the same thing. PTAL.